dist/angular-ui-tree.js

```diff
@@ -1393,7 +1435,7 @@
               angular.element($document).unbind('mouseup', dragEndEvent);
               angular.element($document).unbind('mousemove', dragMoveEvent);
               angular.element($document).unbind('mouseleave', dragCancelEvent);
-              angular.element($document).unbind('keydown', keydownHandler);
+              angular.element($document).unbind('keyup', keyupHandler);
```

Comment:
could you add a comment why the key up handler is needed vs the key down, just in case we try to switch it back to key down in the future